responders/src/UI/embc-responder/src/app/feature-components/search/evacuee-search/evacuee-search-results/evacuee-search-results.component.ts

```diff
@@ -105,15 +119,58 @@ export class EvacueeSearchResultsComponent implements OnInit {
   }
 
   openWizard(): void {
-    this.cacheService.set(
-      'wizardOpenedFrom',
-      '/responder-access/search/evacuee'
-    );
-    this.evacueeSessionService.setWizardType(WizardType.NewRegistration);
+    if (this.evacueeSessionService.paperBased === true) {
+      this.evacueeSearchResultsService
+        .essFileExists(this.paperBasedEssFile)
+        .subscribe({
+          next: (result) => {
```

Comment:
Think about restructuring this code. Maybe call the exists api in the page load and hold the value, that will simplify this method